### PR TITLE
[DEVOPS-511] Build uuu files in jenkins

### PR DESCRIPTION
**Pull request by Jasper-Ben**

### CODE REVIEW

**Review by Eximmius — APPROVED** [review]
This almost doubles the firmware build time, but I can live with that.

**Comment by Eximmius** [comment]
> Do you need the "normal" images for release 2 at all? Or only the uuu images? In the latter case we could replace the build process for release 2 instead of building it in addition

In this stage I would still let it build in addition to the normal files. For development it is good to get everything that you can build out of yocto.

File: buildspecs/build_firmware_images_develop.yml [hunk 1]
```diff
@@ -30,6 +30,12 @@ phases:
     commands:
       # build artifacts
       - kas shell -k -c "bitbake ${images}" kas-irma6-pa.yml
+      # if rel2, build uuu files
+      - >
+        if [ "${MULTI_CONF}" = "imx8mp-evk" ]; then
```
[the remark this answers is not in the document]
Comment (Jasper-Ben) [reply buildspecs/build_firmware_images_develop.yml]: Done.

**Comment by Jasper-Ben** [comment]
> > Do you need the "normal" images for release 2 at all? Or only the uuu images? In the latter case we could replace the build process for release 2 instead of building it in addition
> 
> In this stage I would still let it build in addition to the normal files. For development it is good to get everything that you can build out of yocto.

Alright. This whole process is just temporary and more of a quick fix. I expect this to change with GitLab CI.

**Pull request merged**